book/content/very_large_datasets/very_large_datasets.md

```diff
@@ -0,0 +1,56 @@
+
+After a contributor has uploaded a dataset, the data-sharing infrastructure should relieve data contributors from providing technical support for individual users of the data, while also facilitating discussions/collaborations between data contributors and users. To enable easy and flexible online search and access, datasets should be indexed and tagged using descriptive language that creates clear categories. Users should also be able to easily create visualizations of the data, preferably using tools embedded directly in the infrastructure. 
+
+As time passes and technologies advance and change, it will become important that these data-sharing infrastructures are expandable to include new formats, workflows, and genres of data. Given the current trends in dataset size, it will also be important for data-sharing infrastructures to be able to accommodate larger and larger datasets. 
+
+Ideally, data-sharing infrastructures can help make open research practices more accessible by hosting a list of Frequently Asked Questions, or by providing teaching tools to help new contributors and new users figure out how to use the infrastructure. The FAQ and teaching tools can also bridge gaps between language and ways of thinking that arise between areas of expertise. Discussion forums can provide opportunities for user interactions and collaborations, nurturing an open research community. 
+
+Lastly, a data-sharing infrastructure should have measures in place to prevent unfair or bad faith use of the data. 
+
+ Refs for this section: 
+ - https://gking.harvard.edu/files/gking/files/dvn.pdf
+ - https://crcns.org/files/news/crcns-neuroinformatics-article.pdf
+
+## How should researchers prepare their Very Large Datasets for sharing?
+
+The following is a suggested check-list for meta-data to include when researchers are preparing to share a Very Large Dataset:
+ - Description of the theoretical context and experimental paradigm
+ - Description of experimental conditions, ideally a fully descriptive data collection protocol
+   - date and time of data collection
+   - researchers involved in data collection
+   - species, age, sex, etc of experiment participants/subjects
+   - housing conditions / home context of experiment participants/subjects
+   - tools and techniques used to collect data (electrode type, video camera model, tissue staining technique, microphone type, etc)
+   - surgical procedures involved, if any
+   - if stimuli were used, how were the stimuli generated?
+   - how timings of stimuli, recordings, and any other events in the experiment were correlated and/or synchronized
```

Comment:
Reading this it might be useful to split this sections into general metadata that can apply to all kinds of datasets and an example for a specific dataset e.g. from neuroscience as the references to surgical procedures and stimuli seem very specific.

Reply:
Good point! So it would split into a section for "all kinds of datasets", then multiple sections about what additional meta-data would be good to add for specific kinds of datasets?